pyintegration/tests/test_parquet.py

```diff
@@ -0,0 +1,146 @@
+#
```

Comment:
Just sticking this comment here...
Changes that don't seem to be tested:
1) `column.py`
2) `dtypes.py`
3) `stream/__init__.py`